pkg/frontend/querymiddleware/codec.go

```diff
@@ -297,12 +297,12 @@ func DecodeInstantQueryTimeParams(r *http.Request) (int64, error) {
 }
 
 func DecodeLabelsQueryTimeParams(r *http.Request) (start, end int64, err error) {
```

Comment:
It would be great to also update `DecodeRangeQueryTimeParams` and `DecodeInstantQueryTimeParams` to use the new util as well (among a bunch of other updates to this codec package) - however I am keeping this scope down for this PR since `DecodeLabelsQueryTimeParams` is only used for this new functionality so far, unlike the other two functions.